src/boundingbox.jl

```diff
@@ -26,11 +26,11 @@ BoundingBox(width, height, depth) = BoundingBox(0mm, 0mm, 0mm, width, height, de
 
 isabsolute{P1, P2}(b::BoundingBox{P1, P2}) = isabsolute(b.x0) && isabsolute(b.a)
 
-typealias AbsoluteBox{N}   BoundingBox{NTuple{N, Length{:mm, Float64}},
+@compat const AbsoluteBox{N} = BoundingBox{NTuple{N, Length{:mm, Float64}},
                                        NTuple{N, Length{:mm, Float64}}}
-typealias Absolute2DBox    BoundingBox{Tuple{AbsoluteLength, AbsoluteLength},
+@compat const Absolute2DBox =  BoundingBox{Tuple{AbsoluteLength, AbsoluteLength},
```

Comment:
Yeah, it seems to be working. I'll remove it for all nonparamterized typealiasing calls